specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -79,13 +82,27 @@
             "$ref": "#/parameters/ApiVersionParameter"
           },
           {
-            "$ref": "#/parameters/AutocompleteModeParameter"
+            "name": "autocompleteMode",
+            "in": "query",
+            "required": true,      
+            "type": "string",
+            "enum": [
```

Comment:
There is already an `AutocompleteMode` enum definition later in the file. Can this parameter definition refer to that type instead of re-defining it?

Reply:
I tried that. Doing so somehow makes AutocompleteMode a generic `object` type.

Reply:
Too bad. Doing it this way still results in only one generated `AutocompleteMode` enum though, right?

Reply:
Yes